msmbuilder/featurizer/multichain.py

```diff
@@ -0,0 +1,510 @@
+
+        if calculate_for == 'ligand':
+            self.calculate_for = self.ligand_chain
+        elif calculate_for == 'protein':
+            self.calculate_for = self.protein_chain
+        elif calculate_for == 'custom':
+            if calculate_indices is None:
+                raise ValueError("Please specify custom calculate_indices")
+        else:
+            raise ValueError("Please specify a valid option")
+
+
+        if align_indices is not None:
+            if align_by is not 'custom':
+                if not self._check_indices(self.reference_frame, self.align_by,
+                                           align_indices):
+                    raise ValueError("align_indices must be on the " \
+                                     "align_by chain")
+            else:
+                if not all(align_indices[i] in
+                           range(self.reference_frame.n_atoms)
+                           for i in range(len(align_indices))):
+                    raise ValueError("align_indices must exist")
+            self.align_indices = align_indices
+        else:
+            self.align_indices = self._get_atom_range(self.reference_frame,
+                                                 self.align_by)
+
+        if calculate_indices is not None:
+            if calculate_for is not 'custom':
+                if not self._check_indices(self.reference_frame,
+                                           self.calculate_for,
+                                           calculate_indices):
+                    raise ValueError("calculate_indices must be on the " \
+                                     "calculate_for chain")
+            else:
+                if not all(calculate_indices[i] in
+                           range(self.reference_frame.n_atoms)
+                           for i in range(len(calculate_indices))):
+                    raise ValueError("calculate_indices must exist")
+            self.calculate_indices = calculate_indices
+        else:
+            self.calculate_indices = self._get_atom_range(self.reference_frame,
+                                                     self.calculate_for)
+
+
+    # custom option will never see this
+    def _get_atom_range(self, traj, chain):
+        if chain == self.protein_chain:
+            offset = self.p_atom_offset
+        elif chain == self.ligand_chain:
+            offset = self.l_atom_offset
+        else:
+            raise ValueError("Protein or ligand chain required")
+        atom_range = range(offset,offset+traj.top.chain(chain).n_atoms)
+        return atom_range
+
+
+    # custom option will never see this
+    def _check_indices(self, traj, chain, indices):
+        atom_range = self._get_atom_range(traj, chain)
+        return all(indices[i] in atom_range for i in range(len(indices)))
+
+
+    def _naive_rmsd(self, traj, ref, idx):
+        return np.sqrt(np.sum(np.square(traj.xyz[:,idx,:] - ref.xyz[:,idx,:]),
```

Comment:
Missing a 3 in that `sqrt`: https://github.com/mdtraj/mdtraj/issues/948#issuecomment-162160711

I probably forgot this in my own code